viewer/packages/cad-geometry-loaders/src/CadManager.ts

```diff
@@ -130,9 +130,11 @@ export class CadManager {
     return this._needsRedraw;
   }
 
-  updateCamera(camera: THREE.PerspectiveCamera): void {
-    this._cadModelUpdateHandler.updateCamera(camera);
-    this._needsRedraw = true;
+  updateCamera(camera: THREE.PerspectiveCamera, cameraInMotion: boolean): void {
+    this._cadModelUpdateHandler.updateCamera(camera, cameraInMotion);
+    if (cameraInMotion) {
+      this._needsRedraw = true;
+    }
```

Comment:
If the camera has moved then cadManager will have this flag set already. So I don't believe this is needed.

Reply:
Ah, okey. Just assumed it was necessary since it was there before. Tested it now, and it seems to work fine without setting the flag. Removing.